README.md

```diff
@@ -46,7 +46,6 @@ The EditorConfig Gedit plugin fully supports the following EditorConfig
 [properties][]:
 
 * indent_style
-* end_of_line
```

Comment:
How about

    * end_of_line (only for gedit <= 3.14)

Reply:
That works. Not sure anyone is still running versions that old, but can't hurt. (RHEL/CentOS people probably still are, anyway.)

I'll make that change now.

Reply:
Done.